### ISSUE TITLE
Add/Remove Like to a Post (with post_id / user_id)

### ISSUE THREAD

**Issue by LaRepDuClic** [humhub/humhub-modules-rest#25]
It would be useful to allow this functionnality of adding/removing a like to a post

Example

- POST https://www.humhub.com/api/v1/like/byContent/{id} with data { 'user_id': <userId> }
- DEL https://www.humhub.com/api/v1/like/byContent/{id} with data { 'user_id': <userId> }

**Comment by luke-** [comment]
Yes, it should be realized via a DELETE request.

I can't say when yet. From our side it is not planned in the short term.

**Comment by JjmWeber** [comment]
Is this enhancement still planned ? It will be useful.

**Comment by luke-** [comment]
Yes, but not on the agenda at the moment. Feel free to contribute!

**Comment by JjmWeber** [comment]
Not sure, I have enough skills but I will take a look. Thx
